src/stories/5-Causes.stories.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { Causes } from '../components/Causes';
+
+export default {
+  title: 'Causes',
+  component: Causes,
+  decorators: [withKnobs]
+};
+
+const data = {
+  causes: [
+    {
+      title: 'Raise Funds for Saving Poor Kids',
+      description: 'Aliq is notum hender erit a augue insu image pellen tes.',
+      goal: 10000,
+      raised: 9000,
+      link: {
+        text: 'Donate Now',
+        url: '#readmore',
+        sort: 0
+      },
+      image: {
+        alternativeText: '',
+        url:
+          'https://charity-cms-dev.s3.eu-central-1.amazonaws.com/causes-card2_8563b353ef_b5daf5114b_ace5db76e1.jpeg'
+      },
+      id: '5eab4022f2da718214b6f2cd'
```

Comment:
no need for id

Reply:
Id is required because this is an array and we are looping in it and it's used in key